src/plugins/vis_type_timeseries/server/lib/vis_data/request_processors/series/date_histogram.test.js

```diff
@@ -86,8 +86,7 @@ describe('dateHistogram(req, panel, series)', () => {
             },
           },
           meta: {
-            bucketSize: 10,
-            intervalString: '10s',
```

Comment:
`intervalString` was removed because I could not find a place where we would use it